components/new_post/user_profile.tsx

```diff
@@ -0,0 +1,141 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {ReactNode} from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import Constants from 'utils/constants';
+import {fromAutoResponder} from 'utils/post_utils';
+
+import {Post} from '@mattermost/types/posts';
+
+import BotBadge from 'components/widgets/badges/bot_badge';
+import Badge from 'components/widgets/badges/badge';
+import UserProfile from 'components/user_profile';
+import PostHeaderCustomStatus from 'components/post_view/post_header/post_header_custom_status';
+
+type Props = {
+    post: Post;
+    compactDisplay?: boolean;
+    colorizeUsernames?: boolean;
+    isBusy?: boolean;
```

Comment:
I'm looking at the existing code, and I don't think `isBusy` is actually used any more. I'm pretty sure it goes back to the old pre-Calls, pre-Zoom plugin video calling thing we had that never quite worked